packages/components/radio/src/radio.css

```diff
@@ -0,0 +1,87 @@
+/* General........................................................ */
+
+:host {
+	& label {
+		display: block;
+		position: relative;
+		padding-left: var(--ts-unit-plus);
+		margin: var(--ts-unit-half);
+		cursor: pointer;
+		font-size: var(--ts-fontsize);
+		-webkit-user-select: none;
```

Comment:
We have postcss, no need for browser prefixes.

Reply:
Fixed